form_jsons/config/production.json

```diff
@@ -4,5 +4,6 @@
     "enableCsrf": true,
     "safelist": ["funding-service-design-application-store-dev.london.cloudapps.digital", "funding-service-design-application-store-test.london.cloudapps.digital", "funding-service-design-application-store.apps.internal"],
     "jwtAuthCookieName": "fsd_user_token",
-    "jwtAuthenticationUrl": "https://authenticator.live.fundingservice.co.uk/service/magic-links/invalid"
+    "jwtRedirectToAuthenticationUrl": "https://authenticator2.test.gids.dev/sessions/sign-out",
```

Comment:
As this will vary by environment, I guess we'll need to use an env var in the manifest, rather than baking it in the docker image.

Reply:
Yep good point, was wondering the best way to handle this. We do have a pattern of copying in these config .json files (for development and production at the moment) which are then switched by the global NODE_ENV env var. Should we just make more of these config .json files? Might be an easier way of managing these non-secret settings than having to set lots of env vars in every environment?

Reply:
@harryyo You can set it in the manifest as a regular ENV VAR as per: https://github.com/communitiesuk/digital-form-builder/blob/09986450215c16cd612aae5a5d575177789a2a01/runner/config/custom-environment-variables.json#L48 so no need to rely on `NODE_ENV`

Reply:
@harryyo I'd prefer more in ENV VARs as a rule as it keeps the app more 12-factor-y.

Reply:
@gidsg so you reckon we move all these config vars into the manifest-form-runner.yml and remove this config copying in? Did you manage to get the safelist working with an env var too @Adam-W1 ?

Reply:
@harryyo I am happy for things that don't vary by environment to still be in the config that gets copied in.

Reply:
Done